tools/ldb_cmd.cc

```diff
@@ -2928,8 +2929,12 @@ void MultiGetCommand::DoCommand() {
   bool failed = false;
   for (size_t i = 0; i < num_keys; ++i) {
     if (statuses[i].ok()) {
-      fprintf(stdout, is_value_hex_ ? "0x%s\n" : "%s\n",
+      fprintf(stdout, is_value_hex_ ? "%s%s0x%s\n" : "%s%s%s\n",
+              (is_key_hex_ ? StringToHex(keys_[i]) : keys_[i]).c_str(), DELIM,
               values[i].ToString(is_value_hex_).c_str());
+    } else if (statuses[i].IsNotFound()) {
+      fprintf(stdout, "Key not found: %s\n",
+              (is_key_hex_ ? StringToHex(keys_[i]) : keys_[i]).c_str());
```

Comment:
In the below `else` block, the `failed` flag is supposed to be set to true, right?

Reply:
Yup. If you expand the lines hidden by git, It's there (in line 2942)